spec/spec_helper.rb

```diff
@@ -1,22 +1,23 @@
 $VERBOSE=nil  # Don't care about world writable dir warnings and the like
 
 require 'pathname'
-if File.exist? File.expand_path("../Gemfile", Pathname.new(__FILE__).realpath)
+# we know Gemfile is there, no idea why this causes issues
+# if File.exist? File.expand_path("../Gemfile", Pathname.new(__FILE__).realpath)
   require 'rubygems'
   require 'bundler/setup'
-else
+# else
   # If running on windows, can't use Origen helpers 'till we load it...
-  if RUBY_PLATFORM == 'i386-mingw32'
-    `where origen`.split("\n").find do |match|
-      match =~ /(.*)\\bin\\origen$/
-    end
-    origen_top = $1.gsub("\\", "/")
-  else
-    origen_top = `which origen`.strip.sub("/bin/origen", "")
-  end
-
-  $LOAD_PATH.unshift "#{origen_top}/lib"
-end
+#  if RUBY_PLATFORM == 'i386-mingw32'
+#    `where origen`.split("\n").find do |match|
+#      match =~ /(.*)\\bin\\origen$/
+#    end
+#    origen_top = $1.gsub("\\", "/")
+#  else
+#    origen_top = `which origen`.strip.sub("/bin/origen", "")
+#  end
+#
+#  $LOAD_PATH.unshift "#{origen_top}/lib"
+# end
```

Comment:
So this works on windows now? If so maybe clear out the old comments?

Reply:
That code was checking to see if ../Gemfile exists. If it doesn't exist, it was checking the OS and which origen. None of that code was working correctly from Windows.

It all seems like it became outdated long ago (I don't know of a version of origen_testers that doesn't have Gemfile 1 directory up from spec/)